components/text/gettext/Makefile

```diff
@@ -42,6 +43,12 @@ COMPONENT_LICENSE_FILE=	COPYING
 
 include $(WS_MAKE_RULES)/common.mk
 
+# Our sed misbehaves during the tests and let msgfilter-2 and msgfilter-4 tests being skipped.
```

Comment:
... if our sed misbehaves, then it is a bug in our sed.  I'd expect some reference to an illumos bug then.

Isn't it instead that GNU gettext testing simply uses some sed GNUisms?  If that's the case then maybe testing should be fixed.